client/src/components/ProfileDetails/RequestForm/RequestForm.tsx

```diff
@@ -28,7 +27,6 @@ export default function RequestForm({ sitter }: Props): JSX.Element {
     if (sitter) {
       postRequest(sitter._id, startDate, endDate).then((data) => {
```

Comment:
Data is being defined but never used, please exclude 'data' and address warning.